chain-storage-sqlite-old/Cargo.toml

```diff
@@ -21,5 +21,11 @@ version = "0.16.0"
 features = ["bundled"]
 
 [dev-dependencies]
+criterion = "0.3.0"
```

Comment:
can these new dependencies be included only when `cargo bench` is launch? Just adding less deps that are not use when running the bench?

Reply:
Hm, that's a bit tricky with the current setup. The working solution would be to:

- move `bench`-only dependencies
- mark them optional
- join them under a feature with the name like `with-bench`
- require this feature for `bench` builds

This *should* be fine, but requires activating features with `cargo` flags which don't really work with cargo workspaces right now.

Reply:
Apparently, running that command from within `chain-storage-sqlite-old` works just fine. The workspace is still recognized correctly and the top-level `target` directory is used. That's just a bit inconvenient.